### ISSUE TITLE
Revised time in the Edit Media window does not reflect timezone

### ISSUE THREAD

**Issue by slyfur** [mansj/enable-media-replace#23]
This is an excellent plug-in. It does exactly what we need.  I just noticed that the "Revised" time/date in the Edit Media window is showing (for example) "Revised: January 6, 2017 9:13 pm" on a file that I uploaded at 3:13 pm in the "Chicago" timezone. This might be confusing to some of our less tech-savvy end-users.

Is this something that I can manually override? I'm fairly new to WordPress, PHP and Github.

Thanks!

**Comment by mansj** [comment]
No, because the server will always report the time in the time and time zone set on the server, so unless you can change the time zone there you can't.

**Comment by slyfur** [comment]
I have full control of the server. It is set for the Central (Chicago) time zone. WordPress is set for the same. Page edits and other times in WordPress appear to match up with the local time, but this plug-in does not.

**Comment by kool-kat** [comment]
Perhaps we could be given an (+- HH:MM) offset parameter that could be applied to system time and the calculated value show to users?

We have issues of 'daylight saving time' where time can be +- 1 Hour depending on time of year. Makes for confusion when comparing file datestamps
